pkgs/games/ckan/default.nix

```diff
@@ -28,7 +28,7 @@ stdenv.mkDerivation rec {
       install -m 0644 $exe $out/bin
       makeWrapper ${mono}/bin/mono $out/bin/$(basename $exe .exe) \
         --add-flags $out/bin/$exe \
-        --set LD_LIBRARY_PATH ${gtk.out}/lib
+        --set LD_LIBRARY_PATH ${gtk.out}/lib:${curl.out}/lib
```

Comment:
This may be more convenient as `stdenv.lib.makeLibraryPath [ gtk curl ]`, or does that not work?